cli/proc_state.rs

```diff
@@ -358,6 +402,13 @@ impl ProcState {
       reload: reload_on_watch,
     };
 
+    let maybe_file_watcher_reporter: Option<&dyn deno_graph::source::Reporter> =
```

Comment:
I'm not sure if we should pass this reporter to `create_graph` or to `CliModuleLoader` - the latter will ensure all local files that V8 execute will get watched